engine/interpreter/src/main/scala/pl/touk/nussknacker/engine/spel/Typer.scala

```diff
@@ -269,23 +272,32 @@ private[spel] class Typer(classLoader: ClassLoader, commonSupertypeFinder: Commo
     case Unknown => Valid(Unknown)
     case s: SingleTypingResult =>
       extractSingleProperty(e)(s)
-        .map(Valid(_)).getOrElse(invalid(s"There is no property '${e.getName}' in type: ${s.display}"))
     case TypedUnion(possible) =>
-      val l = possible.toList.flatMap(single => extractSingleProperty(e)(single))
+      val l = possible.toList.flatMap(single => extractSingleProperty(e)(single).toOption)
```

Comment:
can we use some validation method and not convert toOption and vice versa?

Reply:
I'm not sure if I understand your comment... As a user I don't want to see such detailed (accumulated) error that typer can't find property in type A because ..., type B because ..., etc. I just want to have simple message. It is why I skip this error part. It is not a walk-around, it is a part of business logic...

Reply:
yes, yes I understand the idea, but still - I'd expect some fancy method that'll handle that, like .sequence.leftMap(_.head) ;) 
but we can leave it as it is